app/controllers/staff/refunds_controller.rb

```diff
@@ -20,12 +20,12 @@ def create
     authorize @activity
     @refund = RefundForm.new(refund_params)
 
-    return render :new unless @refund.valid?
-
-    CreateRefund.new(
+    refund_created = @refund.valid? && CreateRefund.new(
```

Comment:
How do you feel about applying the same succinct pattern to the `update` method in this same controller?

Reply:
I was busy thinking that we should become un-succinct in the same manner as `CreateActual` for consistency...

Reply:
Too late, I already changed the actuals controller to be more succinct, except James suggested some changes to the succinctness that will be a third way to which we can all converge. ;)

Reply:
Oh, well, alright then.